functions_and_scope_assignment.js

```diff
@@ -0,0 +1,287 @@
+// Your function should take in one parameter: x (the number to count up to)
+// // ****need help*****
+// function multiple (num){
+//   for ( let num =1; num <= 50; i++){
+//     if ( num % 3 ===0 && num % 5 ===o){
+// return"FizzBuzz"
+// } else if (num % 3 ===0){
+//   return "Fizz
+//   for ( let num2 =2; num2 <= 50; i++){
+//     if( i % 3 === 0 && i % 5 === 0){
+//       num2.push("FizzBuzz")
+//     }else if ( i % 3 ===0){
+//       num2.push("Fizz")
+//     }else if ( i %5 ===0){
+//       return " Buzz"
+//       num2.push("Buzz")
+//     } else{
+//       return i
+//     }
+//     num2.push(i)
+//   }
+// }
+// return num2
+//   }
+// }
+//   console.log(multiple(50))
+//===============================================================================
+// ## Question 9
+//
+// Write a function named `reverse` that takes an array as a parameter. The function should return an array in reverse order.
+//
+//
+// Example:
+//
+// Input: `reverse([1, 2, 3])`
+//
+// Output: `[3, 2, 1]`
+// let arr = [1, 2 ,3];
+// const reverse = (arr) => {
+//   let reverseArray = [];
+// for (let value of arr) {
+//   reverseArray.unshift(value);
+// }
+// return reverseArray;
+// }
+// console.log(reverse(arr));
+// ==================================================================
+// ## Question 10*
+//
+// Write a function that prints out the most frequently occurring element in an array.
+//
+// Example:
+//
+// Input: `[2,1,4,2,5,2,3,23,3,3,2,2,2,24,56,2,2,5,46,2]`
+//
+// Output: `2`
+//let arr = [2,1,4,2,5,2,3,23,3,3,2,2,2,24,56,2,2,5,46,2];
+//
+//====================================================================
+// ## Question 11
+//
+// Write a function that sums all the even numbers in an array.
+//
+// Example:
+//
+// Input: `[1,2,5,64,23,22,91,3]`
+//
+// Output: `88`
+//
+//  let arr = [1,2,5,64,23,22,91,3];
+// const even = (arr) => {
+//   let sum = 0
+//  for ( let i = 0; i < arr.length; i ++){
+//     if ( arr[i] % 2 === 0){
+// sum = sum + arr[i]
+//   }
+// }
+// return sum
+// console.log(sum)
+// };
+//  let result = even(arr)
+//  console.log(result);
+//==========================================================================
+// ## Question 12
+//
+// Write a function that flips an object.  All of the keys are now values and all of the values are now keys.
+//
+// Example:
+//
+// Input: `{1: "uno", 5: "cinco"}`
+//
+// Output: `{"uno": 1, "cinco": 5}`
+
+// let object = {1: "uno", 5: "cinco" }
+// const flipObj = (object) =>{
+//   let flip = {}
+//   for ( let key in object) {
+//     flip[object[key]] = key;
+//   }
+//   return flip
+// }
+// console.log(flipObj(object))
+//=========================================================================
+// ## Question 13
+//
+// Write a function that determines if a value is inside of array.
+//
+// Example:
+//
+// Input: `[1,4,6,9,10], 5`
+//
+// Output: `false`
+// let arr = [1,4,6,9,10]
+// const valueArr = ( arr, val) =>{
```

Comment:
I don't think this function is working as intended. It gives me false even when I include a number that is inside the array. Try to figure this out and let me know if you have questions or difficulties.